src/pages/junghyowon/Main/JungMain.js

```diff
@@ -1,7 +1,235 @@
 import React from 'react';
+import { useState } from 'react';
```

Comment:
```suggestion
import React, { useState } from 'react';
```
로그인에서는 잘하셨는데 메인페이지도 똑같이 해주시면 좋을거 같아요